mmdet/models/dense_heads/yolo_head.py

```diff
@@ -488,3 +508,21 @@ def _get_targets_single(self, anchors, responsible_flags, gt_bboxes,
         neg_map[sampling_result.neg_inds] = 1
 
         return target_map, neg_map
+
```

Comment:
Let's probably move this to `mmdetection/tests/` and add some dummy input to formulate it as a unit test.

Reply:
The method `aug_test` is not for software test, but for inference.
Please refer the `aug_test` methods of [AnchorHead](https://github.com/open-mmlab/mmdetection/blob/c8511649550834ea168f610411a47a39cf194767/mmdet/models/dense_heads/anchor_head.py#L676-L692) and [StandardRoIHead](https://github.com/open-mmlab/mmdetection/blob/c8511649550834ea168f610411a47a39cf194767/mmdet/models/roi_heads/standard_roi_head.py#L267-L292).

Reply:
Sorry for my oversight.